Products/GenericSetup/registry.py

```diff
@@ -644,6 +644,10 @@ def __init__(self):
     def getProfileInfo(self, profile_id, for_=None):
         """ See IProfileRegistry.
         """
+        if profile_id.startswith("profile-"):
+            profile_id = profile_id[len('profile-'):]
```

Comment:
For me the `len('profile-')` makes the intent more explicit. When checking whether the code is correct, you immediately see that this does the correct thing. With `len(8)` I would start counting the number of characters in `profile-` to see if those are really 8, and then do a recount to double check it.

At least, that is my reasoning. I don't mind changing it if wanted.